app/src/main/java/com/eveningoutpost/dexdrip/Models/JoH.java

```diff
@@ -190,6 +191,18 @@ public static long msSince(long when) {
         return (tsl() - when);
     }
 
+    public static long msSince(long end, long start) {
```

Comment:
Does this really need a helper function if its just x - y ?

Reply:
Well, I can remove it if this is a problem.